### DISCUSSION TITLE
docs/master/contribution-guide

### DISCUSSION THREAD

**Discussion by giscus** [gorse-io/docs#112] [Giscus]
# docs/master/contribution-guide

Contribution Guide This page will show how to work on the Gorse source code. Setup Develop Environment The following installations are required: Go (&gt;= 1.18): Since Go features from 1.18 are used in Gorse, the version of the compiler must be greater than 1.18. GoLand or Visual Studio Code is highly recommended as the IDE to develop Gorse. Docker Compose: Databases are required for unit tests. It's convenient to manage databases on Docker Compose.

https://gorse.io/docs/master/contribution-guide.html

**Comment by 17020749** [comment]
alo

**Reply by 17020749** [reply]
i got this error when running worker: "score should be the same"

**Reply by 17020749** [reply]
I run completely 
 nodes using golang serive
